ItemList.tsx

```diff
@@ -0,0 +1,96 @@
+import { useState } from 'react';
+import { FlatList, TextInput, TouchableOpacity, View } from 'react-native';
+import { ItemCard } from './ItemCard';
+import { StyleSheet } from 'react-native';
+import { ImageData } from './ItemCard';
+import { Dimensions } from 'react-native';
+
+const numColumns = 3;
+const screenWidth = Dimensions.get('window').width;
+const itemWidth = screenWidth / numColumns;
```

Comment:
Good idea to maximize the screen space available here but you also have to consider padding in between items. This causes the content to overflow and allows for a little horizontal scrolling which is not ideal.